analytics_data_api/v0/tests/test_connections.py

```diff
@@ -33,7 +33,7 @@ def test_signing(self):
         self.assertTrue('my_access_key' in auth_header)
 
     def test_timeout(self):
-        def fake_connection(_address):
+        def fake_connection(_address, _timeout):
```

Comment:
I also encountered this when upgrading boto on master. This will conflict because I replaced the parameters to this function with `*args`. This also works, so feel free to keep these changes when resolving the conflict.